packages/docs-app/src/examples/core-examples/navbarExample.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import * as React from "react";
+
+import { Navbar, NavbarDivider, NavbarGroup, NavbarHeading, Switch } from "@blueprintjs/core";
+import { BaseExample, handleBooleanChange } from "@blueprintjs/docs";
+
+export interface INavbarExampleState {
+    alignRight?: boolean;
+}
+
+export class NavbarExample extends BaseExample<INavbarExampleState> {
+    public state: INavbarExampleState = {
+        alignRight: false,
+    };
+
+    private handleAlignRightChange = handleBooleanChange(alignRight => this.setState({ alignRight }));
+
+    protected renderExample() {
+        return (
+            <Navbar>
+                <div style={{ margin: "0 auto", width: 480 }}>
```

Comment:
wait so why is this required? it does not appear in the docs code sample.

I think it should be a feature of `Navbar` itself, like `<Navbar innerWidth={480}>`

Reply:
Fixed via a simple `width: 100%` rule on the wrapping element.